source/test/TestApp/TestApp.cpp

```diff
@@ -104,6 +107,10 @@ namespace PlayFabUnit
         testRunner.Add(pfMultiUserInstanceTest);
 #endif
 
+        PlayFabQoSTest pfQosTest;
```

Comment:
I believe that QoS is exclusively useful for clients. It may only be worth keeping this inside of the
```#if !defined(DISABLE_PLAYFABCLIENT_API)```
block

Reply:
While limiting the platforms is good, I don't think this should run from servers either.
Please move this two lines up, into the client-only #if block, so that it's only run in client-only builds.